src/libraries/Microsoft.Extensions.Configuration/src/ConfigurationManager.cs

```diff
@@ -66,7 +66,8 @@ public ConfigurationManager()
 
         IDictionary<string, object> IConfigurationBuilder.Properties => _properties;
 
-        IList<IConfigurationSource> IConfigurationBuilder.Sources => _sources;
+        /// <inheritdoc />
+        public IList<IConfigurationSource> Sources => _sources;
```

Comment:
Can you add a test to ensure clearing works as expected? That was the scenario reported in the original issue.

Reply:
That's now done